src/screens/Home/InvestmentsSection.tsx

```diff
@@ -53,16 +53,24 @@ import Text from 'components/core/Text';
 import TokenIcon from 'components/display/TokenIcon';
 import CategoryListItem from './components/CategoryListItem';
 
+// Types
+import type { Theme, ColorsByThemeProps } from 'models/Theme';
+
 type IInvestmentsSection = {};
 
 const InvestmentsSection: FC<IInvestmentsSection> = () => {
   const { t } = useTranslationWithPrefix('home.investments');
   const { t: tStaking } = useTranslationWithPrefix('plrStaking');
 
   const navigation = useNavigation();
-  const theme = useTheme();
+  const theme: Theme = useTheme();
   const accounts = useAccounts();
 
+
+  console.log('====================================');
+  console.log('theme', theme);
+  console.log('====================================');
```

Comment:
Please remove the console logs please, or use a logger

Reply:
Removed it.